src/plugins/core/cell.ts

```diff
@@ -286,6 +286,24 @@ export class CellPlugin extends CorePlugin<CoreState> implements CoreState {
 
   exportForExcel(data: ExcelWorkbookData) {
     this.export(data);
+    const incompatible: string[] = [];
+    for (const formatId in data.formats || []) {
+      if (!isExcelCompatible(data.formats[formatId])) {
```

Comment:
We could also "convert" incompatible formats (remove the incompatible parts). That was a format ""Q1 2023" would become "2023" which is a bit better than no format at all. I do similar stuff in the xlsx import. Not 100% sur if it's worth.

Reply:
IMHO, no format at all is better than removing part. It will be easier for the user to catch that a date has no format than just the year. So I think it's not worth.